docs/Decentraland Protocol for Explorers.md

```diff
@@ -0,0 +1,143 @@
+
+#### DAO
+
+Historically, there was many implementations of governance apps for Decentraland. Starting with [Agora][] (off-chain signatures), then Aragon and lastly a [snapshot-based](https://snapshot.org/#/snapshot.dcl.eth). Nowadays, the governance process happens entirely in the [Governance App][dao] and on-chain proposals are enacted using Aragon by the [DAO Committee](https://governance.decentraland.org/transparency). Their principal responsibility is to enact binding proposals on-chain like listing Point of Interests, sending Grants, and any other operations involving the DAO's smart contracts.
+
+### Content layer
+
+The Decentraland Explorer and other applications render representations of the assets from the consensus layer. The content layer defines the mechanisms in which the content is:
+
+1. **Signed**, **validated** and **uploaded** to a node
+2. **Replicated** to other nodes and
+3. **Served** to the final users
+
+To do so, a [content-server](https://github.com/decentraland/catalyst) was created. It is a service that runs inside a Catalyst.
+
+> A _catalyst node_ is a bundle of services that self-contain a copy of Decentraland. The DAO [governs a curated list of trusted catalysts](https://governance.decentraland.org/?type=catalyst) that hold a working copy of Decentraland. It contains everything necessary to make the Explorer work.
+
+The content server stores and synchronizes _Entities_.
+
+An [Entity](https://github.com/decentraland/common-schemas/blob/be7213b40a2180a9a99035eb87e8a5d4b8438e7f/src/platform/entity.ts#L21-L37) is a signed data structure holding a list of content files, a deployment date and pointers. A pointer is a human readable "shortcut" to the entity i.e. the [`0,0`](https://peer.decentraland.org/content/entities/scenes?pointer=0,0) representation of a LAND. Every time an entity is deployed and accepted by the network, the pointers for that entity will now point to the newest entity for that set of pointers. That is the mechanism used to change the content of the land, wearables and emotes.
+
+Unlike LAND, wearables and emotes have [URN pointers](https://github.com/decentraland/urn-resolver/blob/b11aeb677e06e1a9e1d7994efa98a5f11867f854/test/urn.spec.ts#L138-L147). URN are used to reference any asset inside Decentraland, the technology was selected in pursue of a common identifier that enables [extensibility to other networks or remote assets](https://github.com/decentraland/urn-resolver/blob/b11aeb677e06e1a9e1d7994efa98a5f11867f854/test/urn.spec.ts#L269-L280) and [interoperability](https://github.com/common-metaverse/urn-namespaces) with other platforms.
+
+> TODO: Describe entities format
+
+> TODO: Describe entities content restrictions
+
+> TODO: Describe entities IPFS CIDs
+
+> TODO: Describe AuthChain
+
+#### Endpoints that are part of the protocol
```

Comment:
Should the endpoints for sync be here too? (`/pointer-changes `& `/snapshots`)

Reply:
From our last call, I think we can formalize the Content Server Synchronization Protocol in a different document to scope this one in the explorer